### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by francksefu**
Add logo, content, and font family in All HTML files
Add the part Featured Speakers dynamically by using JavaScript
Make the whole HTML file responsive

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,118 @@
...
+> I love work, solve problems, and I have big dreams, maybe in future I will add some AI in this project, that will be possible if you give me strength
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+> I wanna give thank to Microverse
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+This project is [CreativeCommons](https://creativecommons.org/licenses/by-nc/4.0/legalcode) licensed.
+
```
Comment (mwanawabangona) [inline README.md]: - [ ] `required`
- [ ] I suggest that here you add the license and remove the acknowledgments 🥂

File: README.md [hunk 2]
```diff
@@ -1 +1,118 @@
...
+Feel free to check the [issues page](https://github.com/francksefu/Capstone-project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- SUPPORT -->
+## ⭐️ Show your support <a name="support"></a>
+> I love work, solve problems, and I have big dreams, maybe in future I will add some AI in this project, that will be possible if you give me strength
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+> I wanna give thank to Microverse
```
Comment (mwanawabangona) [inline README.md]: - [ ] `required`
- [ ] Here I suggest you can even add the `Creative Commons` and `CindyShine` for acknowledgment as the requirements say.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+      <h1 class="ms-3 mb-4 rouge display-4 margeB">Open house BICC 2022</h1>
+      <div class="pWidth">
+        <p class="text-start p-3 me-3 border border-white border-2 ms-3">
+            On this wonderful day, dear guests, you will have the opportunity to discover all our works with explanation
+          </p>
+      </div>
+      <div class="margeB">
+        <p class="h2 ms-3">2023.10.15(THU) ~ 16(FRI)</p>
+        <p class="mb-5 ms-3 me-3">@Architectural design, sizing and study and more</p>
+
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required`
- [ ] Amazing job so far! 👏 
- [ ] however, there few things you need to address
- [ ] The welcome headers and need to be moved under the logo.
- [ ] Also please try to make the `Oen house bicc` bigger like the template.
- [ ] Plese see the template below

`your template`
image


`original design`
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+                  <p class="col-7 bg-transparent gry">Listen to the message from the general manager who will talk about the history of their company</p>
+                </div>
+            </div>
+            <div class="col-md-2 grytransparent mt-2 mb-2">
+              <div class="row bg-transparent">
+                <!--<i class="icofont-engineer col-2 hauteur bg-transparent text-white fs-1"></i>
+                <span class="material-symbols-outlined text-white hauteur bg-transparent fs-1 col-2">
+                  engineering
+                  </span>-->
+                  <span class="material-icons text-white hauteur bg-transparent fs-1 col-2">
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required`
- [ ] Please the original background just the origin template
- [ ] please see screenshots below for further verification.

`your template`
image

`original template`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @francksefu ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @francksefu, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Good UI design ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**